#!/usr/bin/env python
import roslib;roslib.load_manifest('squirrel_kclhand')

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("model", str_t, 0, "Autogenerated parameter based on model.", "SQUIRREL Three Fingered Hand")
gen.add("Simulated_Joint_Pos_Gain", double_t, 0, "Autogenerated parameter based on model.", 0.1, -100, 100)
gen.add("Simulated_Joint_Inertia", double_t, 0, "Autogenerated parameter based on model.", 1.0, -100, 100)
gen.add("Simulation_Sampling_Time", double_t, 0, "Autogenerated parameter based on model.", 0.02, -100, 100)
gen.add("Simulated_Joint_Damping", double_t, 0, "Autogenerated parameter based on model.", 5.0, -100, 100)
gen.add("Hand_IP_Address", str_t, 0, "Autogenerated parameter based on model.", "127.0.0.1")
gen.add("Hand_Port", int_t, 0, "Autogenerated parameter based on model.", 55200, -100, 100)

exit(gen.generate("squirrel_kclhand", "squirrel_kclhand", "kclhand"))
